src/etc/inc/plugins.inc.d/ntpd.inc

```diff
@@ -396,7 +396,7 @@ function ntpd_configure_do($verbose = false)
     if (empty($config['ntpd']['nomodify'])) { /*note: this one works backwards */
         $ntpaccess .= ' nomodify';
     }
-    if (!empty($config['ntpd']['noquery'])) {
+    if (empty($config['ntpd']['noquery2'])) { /*note: this one works backwards */
```

Comment:
looking back this could probably be `query` for the sake of being entirely correct

Reply:
Not from the config end. Let me merge as is then and deal with that later.